src/components/Login/Login.test.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Login from '.';
+
+describe('Login', () => {
+  it('must render Login and get text Username', () => {
+    render(<Login />);
+    const usr = screen.getAllByText(/Username/);
```

Comment:
```suggestion
    const usr = screen.getByText(/Username/);
```
I think the `getBy*` works in this case